api/src/main/java/org/pmiops/workbench/db/dao/NotebookCohortCacheServiceImpl.java

```diff
@@ -0,0 +1,124 @@
+package org.pmiops.workbench.db.dao;
```

Comment:
Test?

Aside from the obvious, it would also be good to cover deletion of a non-existent cache entry.